src/main/webapp/js/playGame.js

```diff
@@ -357,22 +359,49 @@ function updateUserProgress() {
   fetch(request);
 }
 
+/**
+* Creates a stylized button element that is used for the minimap controls (zoom and map view).
+* @param text the text that should go in the button.
+* @return {Element} an HTML element of the button.
+*/
+function createMinimapButton(text) {
+  const button = document.createElement("div");
+  button.style.backgroundColor = "#fff";
```

Comment:
Rather than having all this style in the JS can you create an ID or class in the styles.css and the give this element that ID or style?